drivers/counter/Kconfig.gecko

```diff
@@ -0,0 +1,14 @@
+# Kconfig - Silicon Labs Gecko Counter driver config
+#
+# Copyright (c) 2019, Piotr Mienkowski
+#
+# SPDX-License-Identifier: Apache-2.0
+
+config COUNTER_GECKO_RTCC
```

Comment:
There is one issue which I didn't addressed in this PR. The `COUNTER_GECKO_RTCC` Kconfig option is always visible even if RTCC module is not present on all gecko SoCs. E.g. is missing on efm32hg. I could make this option depend on `SOC_GECKO_HAS_RTCC` which only some SoCs would select. This however is introducing yet another Kconfig option to maintain. Alternatively I could use the new DT scripting capabilities and do
```
	depends on $(dt_int_val,DT_COMPAT_SILABS_GECKO_RTCC) > 0
```
which would make the option visible only if it's enabled in DT.